tests/Tests.AzureAppConfiguration/RefreshTests.cs

```diff
@@ -1103,15 +1104,15 @@ static void optionsInitializer(AzureAppConfigurationOptions options)
                 .AddAzureAppConfiguration(optionsInitializer, optional: true)
                 .Build();
 
-            IConfigurationRefresherProvider refresherProvider = new AzureAppConfigurationRefresherProvider(configuration);
+            IConfigurationRefresherProvider refresherProvider = new AzureAppConfigurationRefresherProvider(configuration, NullLoggerFactory.Instance);
```

Comment:
If doing multi-line params it's usually better to put all params on new line otherwise the first param ends up as an odd one out.